docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,69 +1,269 @@
-[[get-started-docker]]
-== Running the {stack} on Docker
+[[get-started-stack-docker]]
+== Running the {stack} ("ELK") on Docker
+++++
+<titleabbrev>Running the {stack} on Docker</titleabbrev>
+++++
 
 The Elastic Docker registry contains Docker images for all the products in
 the {stack}: https://www.docker.elastic.co/.
 
-[float]
-[[run-stack-docker]]
-=== Run with Docker Compose
+You can <<run-docker-secure,start the {stack} in Docker>> with security enabled
+and configured by default. This option is great for quickly getting started with
+{es} and {kib} for testing and development.
+
+You can also <<get-started-docker-tls,start the {stack} with Docker Compose>> to
+create a secured, multi-node cluster with a connected {kib} instance. This
+option requires more manual steps, but results in a more robust cluster.
+
+[[run-docker-secure]]
+[discrete]
+=== Start {es} and {kib} in Docker
+Starting in {es} 8.0, security is enabled by default. With security enabled,
+{stack} {security-features} require TLS encryption for the transport networking
+layer, or your cluster will fail to start.
```

Comment:
The TLS comment makes me think as a reader that I need to so something extra because security is enabled. I think this wording made sense in 7.x but maybe not so now ? It is true, but TLS will be enable by default if users follow what we write here so maybe we shouldn't point it out in that manner in the first paragraph ?

Reply:
Good point. I think that we can delete the second sentence and just say:
>Starting in {es} 8.0, security is enabled and configured by default.